src/amo/pages/Home/index.js

```diff
@@ -228,8 +228,12 @@ export class HomeBase extends React.Component {
       return null;
     };
 
+    const showHero =
```

Comment:
You can use `showHero` for this now:

https://github.com/mozilla/addons-frontend/blob/43289081b430f92f9a4b0d0d4956c8882cd1f951/src/amo/pages/Home/index.js#L261-L267